core/util/src/main/scala/net/liftweb/util/IoHelpers.scala

```diff
@@ -85,7 +86,8 @@ trait IoHelpers {
   /**
    * Read an entire file into an Array[Byte]
    */
-  def readWholeFile(file: File): Array[Byte] = readWholeStream(new FileInputStream(file))
+  def readWholeFile(file: File): Array[Byte] = readWholeStream(Files.newInputStream(file.toPath))
+  def readWholeFile(path: Path): Array[Byte] = readWholeStream(Files.newInputStream(path))
```

Comment:
Will the scaladoc appear for both methods if they're grouped like this or should we copy it to be above the second instance, too?